src/components/action-list/action-list.scss

```diff
@@ -106,4 +106,13 @@
   &--no-wrap {
     flex-wrap: nowrap;
   }
+
+  &--vertical {
+    flex-direction: column;
+    align-items: flex-start;
+
+    > .sg-actions-list__hole:not(:last-child) {
```

Comment:
Now I'm wondering if we should really introduce this rule, ona can always add spacing directly to list hole.